src/SFML/Window/Unix/WindowImplX11.hpp

```diff
@@ -265,7 +274,8 @@ class WindowImplX11 : public WindowImpl
     XIC        m_inputContext;   ///< Input context used to get unicode input in our window
     bool       m_isExternal;     ///< Tell whether the window has been created externally or by SFML
     int        m_oldVideoMode;   ///< Video mode in use before we switch to fullscreen
-    Cursor     m_hiddenCursor;   ///< As X11 doesn't provide cursor hidding, we must create a transparent one
+    ::Cursor   m_hiddenCursor;   ///< As X11 doesn't provide cursor hidding, we must create a transparent one
+    ::Cursor   m_lastCursor;     ///< Last cursor used -- this data is not owned by the window and is required to be always valid
```

Comment:
To avoid a clash with `sf::Cursor` and X11 cursor, which lives in the global namespace.